src/main/scala/rocket/PTW.scala

```diff
@@ -26,11 +26,15 @@ class PTWReq(implicit p: Parameters) extends CoreBundle()(p) {
 class PTWResp(implicit p: Parameters) extends CoreBundle()(p) {
   val ae_ptw = Bool()
   val ae_final = Bool()
+  /**page fault  */
```

Comment:
please take care of the format issue.
```suggestion
  /** page fault  */
```